workingtitle-project-cj4.xml

```diff
@@ -5,5 +5,6 @@
 		<Package>PackageDefinitions\workingtitle-vcockpits-instruments-cj4.xml</Package>
 		<Package>PackageDefinitions\workingtitle-vcockpits-instruments-airliners.xml</Package>
 		<Package>PackageDefinitions\workingtitle-aircraft-cj4.xml</Package>
+		<Package>PackageDefinitions\workingtitle-base-aircraft-common.xml</Package>
```

Comment:
It looks like you pulled this package, but maybe left this line still here by accident.

Reply:
Ok, I'll render these points.